geometry_msgs/Pose[] goals
---
bool success
float64 distance
